benchmark/scripts/Benchmark_Driver

```diff
@@ -346,17 +350,40 @@ def positive_int(value):
 
 
 def main():
-    parser = argparse.ArgumentParser(description='Swift benchmarks driver')
-    subparsers = parser.add_subparsers()
-
-    submit_parser = subparsers.add_parser(
-        'submit',
-        help='run benchmarks and submit results to LNT')
-    submit_parser.add_argument(
+    parser = argparse.ArgumentParser(
+        epilog='Example: ./Benchmark_Driver run -i 5 -f Array'
+    )
+    subparsers = parser.add_subparsers(
+        title='Swift benchmark driver commands',
+        help='See COMMAND -h for additional arguments', metavar='<command>')
+
+    parent_parser = argparse.ArgumentParser(add_help=False)
+    benchmarks_group = parent_parser.add_mutually_exclusive_group()
+    benchmarks_group.add_argument(
+        'benchmark',
```

Comment:
Since this is a list, can you change the destination to benchmarks? IIRC the argument is "dest".

So the user will still:

```
--benchmark foo --benchmark bar
```

But programmatically we will see:

```
args.benchmarks
```